File: OPENAPI-REST-API/swagger-client/scala-akka-http-server/src/main/scala/io/swagger/server/model/ServerBulkIpmiPowerResponse.scala
package io.swagger.server.model


/**
 * = Root Type for ServerBulkIpmiPowerResponse =
 *
 * Per-server IPMI power-status results for a bulk lookup.
 *
 * @param results 
 */
case class ServerBulkIpmiPowerResponse (
  results: List[ServerBulkIpmiPowerResponse_results]
)

File: OPENAPI-REST-API/openapi-client/javascript-closure-angular/API/Client/ServerBulkIpmiPowerResponse.js
goog.provide('API.Client.ServerBulkIpmiPowerResponse');

/**
 * Per-server IPMI power-status results for a bulk lookup.
 * @record
 */
API.Client.ServerBulkIpmiPowerResponse = function() {}

/**
 * @type {!Array<!API.Client.ServerBulkIpmiPowerResponse_results_inner>}
 * @export
 */
API.Client.ServerBulkIpmiPowerResponse.prototype.results;

